packages/accounts-controller/src/AccountsController.test.ts

```diff
@@ -191,24 +197,36 @@ function createExpectedInternalAccount({
   lastSelected?: number;
   nameLastUpdatedAt?: number;
 }): InternalAccount {
-  const accountTypeToMethods = {
-    [`${EthAccountType.Eoa}`]: [...Object.values(ETH_EOA_METHODS)],
+  const accountTypeToInfo: Record<
+    string,
+    { methods: string[]; scopes: string[] }
+  > = {
+    [`${EthAccountType.Eoa}`]: {
+      methods: [...Object.values(ETH_EOA_METHODS)],
+      scopes: [EthScopes.Namespace],
+    },
     // TODO: Either fix this lint violation or explain why it's necessary to ignore.
     // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-    [`${EthAccountType.Erc4337}`]: [...Object.values(ETH_ERC_4337_METHODS)],
+    [`${EthAccountType.Erc4337}`]: {
+      methods: [...Object.values(ETH_ERC_4337_METHODS)],
+      scopes: [EthScopes.Mainnet], // Assuming we are using mainnet for those Smart Accounts
+    },
     // TODO: Either fix this lint violation or explain why it's necessary to ignore.
     // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-    [`${BtcAccountType.P2wpkh}`]: [...Object.values(BtcMethod)],
+    [`${BtcAccountType.P2wpkh}`]: {
```

Comment:
Should we take the opportunity to test that we support all account types for Bitcoin in this PR?